ssh/server/handler/ssh.go

```diff
@@ -315,6 +316,32 @@ func shell(api internalclient.Client, sess *session.Session, agent *gossh.Sessio
 		return err
 	}
 
+	// Writes the welcome message. The message consists of a static string about the device
+	// and an optional custom string provided by the user.
+	if target := metadata.RestoreTarget(sess.Client.Context()); target != nil {
+		deviceIdentifier := ""
+
+		if target.IsSSHID() {
+			deviceIdentifier = target.Username + "@" + target.Data
+		} else {
+			deviceIdentifier = target.Username + "@" + target.Data[:7]
+		}
+
+		_, _ = sess.Client.Write([]byte(
+			"Connected to " + deviceIdentifier + " via Shellhub.\n",
+		))
+
+	}
+
+	announcement, err := sess.ConnectionAnnouncement()
+	if err != nil {
+		log.WithError(err).Warn("unable to retrieve the namespace's connection announcement")
+	} else if announcement != "" {
+		sess.Client.Write([]byte("Announcement:\n"))
+		// Indent the announcement by four spaces
+		sess.Client.Write([]byte("    " + strings.ReplaceAll(announcement, "\n", "\n    ") + "\n")) // nolint: errcheck
```

Comment:
You must remove extra spaces and new lines at the left, right, and end of the string.